### PR TITLE
React Capstone Project - Metrics Webapp

### PR DESCRIPTION
**Pull request by meryemsanem**
### COUNTRIES CAPSTONE PROJECT

**This project consists of two main pages:**
- Homepage
- Details

**Homepage :**
When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
Along with the category name, you will display some numeric values per category that come from the API.
When the user clicks (or taps) on a category item, the application navigates to the details page.

**Details:** 
In the details page, the webapp retrieves data from the API to show detailed data in the category.
When the user clicks on the "Back" button (<), the user navigates to the home page.

-------------------------------------------------------------------------------------------------------------------------------------

- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should be stored in the Redux store.
- You should filter the data that you retrieve from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- Write tests for the components and the pages.

### CODE REVIEW

File: src/components/Country.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
```
Comment (Strangeal) [inline src/components/Country.js:2]: - [ ] Excellent job in constructing your capstone project. Your progress thus far is truly commendable. However, it's imperative to refrain from deactivating any `linter` rules. These rules are designed to assist you in writing code that aligns with professional and industry-leading standards.

I kindly urge you to address the `linter` errors present. Rectifying these issues will significantly contribute to upholding a standardized codebase. Happy coding! 😊🚀

File: src/redux/store.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { configureStore } from '@reduxjs/toolkit';
```
Comment (Strangeal) [inline src/redux/store.js:2]: - [ ] Excellent job in constructing your capstone project. Your progress thus far is truly commendable. However, it's imperative to refrain from deactivating any `linter` rules. These rules are designed to assist you in writing code that aligns with professional and industry-leading standards.

I kindly urge you to address the `linter` errors present. Rectifying these issues will significantly contribute to upholding a standardized codebase. Happy coding! 😊🚀

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @meryemsanem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Pixel Perfect Design ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Successfully deployed 🚀 ✅ 
- Used `gh-pages` for deployment ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/styles/Homepage.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  background-color: #355087;
+  justify-content: space-evenly;
+}
+
+.countries-container {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  justify-content: center;
+  width: 100%;
+}
```
Comment (danmainah) [inline src/pages/styles/Homepage.css:22]: - [ ] [optional] @meryemsanem Great work implementing the changes requested, however, I would recommend you stick with the initial design of having 2 countries per row to match the design given, also the details page would match better if the background covered the entire content in the countries details 
image

**Review by danmainah — APPROVED** [review]
Hi @meryemsanem , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Implemented the change requested to ensure the design color matches the design given
✔️  Fixed the linter errors instead of disabling them ensuring the use of quality code.

## Optional suggestions

- [ ] @meryemsanem I noted you used the wrong Gitflow, which is only used when doing group projects, kindly consider always using the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**